_posts/2022-12-09-cyclonedx-maven-plugin-adventure.md

```diff
@@ -0,0 +1,332 @@
+---
+title: "An Adventure with the CycloneDX Maven Plugin"
+date: 2022-12-09
+author: Kevin Conner
+---
+
+# Part One - In the Beginning
```

Comment:
Not sure if you're seeing the rendering locally, but the index page is gonna use the first para of your post as an excerpt, and in this case it's using this heading as an excerpt, which is dumb. Can you add an introductory para above this heading that would read better as an excerpt on the index? Thanks!

Reply:
@jcrossley3 Thanks for that, I've added an introductory paragraph which I hope works.  Let me know what you think.

Reply:
Perfect, thanks!